hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/utils/CheckedConsumer2E.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.mono.state.virtual.utils;
+
+public interface CheckedConsumer2E<T, U, E extends Exception> {
```

Comment:
Please add javadoc, explaining the difference between this interface and `java.util.function.BiConsumer`. I assume the difference is in the fact that it throws checked exception. Is my assumption correct?

Reply:
Correct. I'll rename the class as Joseph proposed and will add some JavaDoc.